escalate/core/views/model_tag.py

```diff
@@ -33,7 +33,7 @@ def post(self, request, *args, **kwargs):
             #above url not being redirected to in super.post so I brute-forced redirect
             return redirect(f'/{model_name}/{model_pk}')
```

Comment:
I tired changing the success url when the cancel button is clicked, but when I tested it, the url changed, but I was redirect back to the same page. I was wondering if there are any ideas about why this happens/how to fix it.